packages/common-common/src/rabbitmq/scripts/publishRmqMessage.ts

```diff
@@ -0,0 +1,34 @@
+import { publishRmqMsg } from "common-common/src/rabbitmq/util";
+import {EventKind, ITransfer, IEventData} from "chain-events/src/chains/aave/types";
+import {CWEvent, SupportedNetwork} from "chain-events/src";
+import {v4 as uuidv4} from 'uuid';
+import {RascalExchanges, RascalRoutingKeys} from "common-common/src/rabbitmq";
+import {RABBITMQ_API_URI} from "commonwealth/server/config";
+
+async function publishRmqMessage() {
```

Comment:
I think its a good idea to try and keep our function names unique. The duplicate name `publishRmqMessage` and the same named import, called on line 24, made me do a double take. 

Perhaps you could rename this function `handlePublishRmqMessage` and keep your imported one `publishRmqMessage`? Or you could rename to something else

Reply:
agreed wasn't sure what to name this and didn't really care cause it was just script. Renamed to `publishRmqMessageScript`